WcaOnRails/app/models/user.rb

```diff
@@ -608,8 +608,7 @@ def self.search(query, params: {})
     users.order(:name)
   end
 
-  attr_accessor :doorkeeper_token
-  def serializable_hash(_options = nil)
+  def serializable_hash(options = nil)
```

Comment:
Just FYI getting rid of the `doorkeeper_token` here was a great idea. I'm going to move this change over into my ongoing PR 1481.